### PR TITLE
Update release notes for Foreman 3.8 GA

### PR DESCRIPTION
**Pull request by Griffin-Sullivan**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

File: guides/doc-Release_Notes/topics/foreman-3.8.0.adoc [hunk 1]
```diff
@@ -4,142 +4,163 @@ A full list of changes is available on https://projects.theforeman.org/issues?se
...
-* Add "last_checkin" attribute to Entitlements Template - https://projects.theforeman.org/issues/30690[#30690]
+* pass:[Some events are not visible even being triggered] - https://projects.theforeman.org/issues/36796[#36796]
+* pass:[CVE-2022-4130: Blind SSRF via Referer header] - https://projects.theforeman.org/issues/36768[#36768]
+* pass:[Add "last_checkin" attribute to Entitlements Template] - https://projects.theforeman.org/issues/30690[#30690]
 
 === API
 
-* GraphQL: Incorrect totalCount when querying with first parameter - https://projects.theforeman.org/issues/36509[#36509]
-* Unable to access API using non-admin users. - https://projects.theforeman.org/issues/36449[#36449]
+* pass:[GraphQL: Incorrect totalCount when querying with first parameter] - https://projects.theforeman.org/issues/36509[#36509]
```
Comment (maximiliankolb) [inline guides/doc-Release_Notes/topics/foreman-3.8.0.adoc:13]: What is `pass:[...]` for?

Comment (ekohl) [reply guides/doc-Release_Notes/topics/foreman-3.8.0.adoc:13]: https://github.com/theforeman/foreman-documentation/pull/2427

**Comment by ekohl** [comment]
Looks like we cherry picked https://github.com/theforeman/foreman-documentation/pull/2427 but not https://github.com/theforeman/foreman-documentation/pull/2437.

**Comment by ekohl** [comment]
I picked it, so if you rebase it I expect it to pass. I still want to add a headline feature before we announce it, but that can be done in a separate PR.

**Comment by ekohl** [comment]
> I still want to add a headline feature before we announce it, but that can be done in a separate PR.

That's to describe https://github.com/theforeman/foreman-installer/pull/878.

**Comment by ekohl** [comment]
Another pick I did, so again: please rebase.

**Review by ekohl — CHANGES_REQUESTED** [review]
I now see `We’d like to thank the following people who contributed to the Foreman {{page.version}} release:` in the contributors section. That's a direct copy from our other repo. Here you need to use `{ProjectVersion}` as the macro.

Other than that I think this is ready.

**Review by ekohl — APPROVED** [review]

**Pull request merged**